### PR TITLE
Allow more sensible input options for enabling animations.

### PR DESCRIPTION
**Pull request by The-Briel-Deal**
Original Issue: https://github.com/hyprwm/Hyprland/issues/5500

#### Describe your PR, what does it fix/add?

**NOTE**: This is my first contribution to Hyprland! I was looking through GitHub issues to look for a good beginner task. If this should be done differently please let me know, I mostly just wanted to get to know the repo so I can contribute more in the future.

The originally reported issue was asking for the Animation config to use a more sensible input for the On/Off field in the animation config. Currently you specify whether or not the feature is enabled as 1 or 0, the reporter and I feel that a clear boolean input is more sensible. That said I did not want to break existing configs. So all I did here was add the option to specify this flag as "on" or "off" and "true" or "false".

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
This should have little to no performance impact. I tested this many times with all valid inputs. And tested this with invalid inputs and behavior is always as expected.

The only thing I wonder if I should change is the amount of comments. Let me know if I should shorten them.

#### Is it ready for merging, or does it need work?
This is ready and manually tested by myself. Please verify my tests however.

### CODE REVIEW

File: src/config/ConfigManager.cpp [hunk 1]
```diff
@@ -1780,10 +1780,14 @@ std::optional<std::string> CConfigManager::handleAnimation(const std::string& co
...
     PANIM->second.pValues    = &PANIM->second;
 
-    // on/off
-    PANIM->second.internalEnabled = ARGS[1] == "1";
-
-    if (ARGS[1] != "0" && ARGS[1] != "1")
+    // If user specifies on/off as "on", "true", or "1" then the animation is enabled.
+    // If user specifies on/off as "off", "false", or "0", then the animation is disabled.
+    // Otherwise, we will let the user know that they specified an invalid on/off state.
+    if (ARGS[1] == "on" || ARGS[1] == "true" || ARGS[1] == "1")
```
Comment (vaxerski) [inline src/config/ConfigManager.cpp:1786]: you can use `configStringToInt`, it handles all of these cases and more.

`PANIM->second.internalEnabled = configStringToInt(ARGS[1]);`

Comment (The-Briel-Deal) [reply src/config/ConfigManager.cpp:1786]: Ty! Implementing this now!

Comment (The-Briel-Deal) [reply src/config/ConfigManager.cpp]: Implemented! Ty for the suggestion. Lmk if you have further thoughts. Doing some testing now.

**Review by vaxerski — APPROVED** [review]
lgtm, thanks!

**Pull request merged**